src/index.html

```diff
@@ -11,12 +11,96 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="image">
+        <a
+          href="https://s3-alpha-sig.figma.com/img/833f/9aab/47d660efd384318772b6b8040db831aa?Expires=1734912000&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=dswIrz-CM71mAanXGvKRXPjecjfgsdMxLMX42Ga2sG0CuUJJq4Jy5x5rODTPEzbqmmsgmlLLJss2~eQyfosxdUyGRzIGLeAOMJ13tFeyrIB5YiQQRtx~9LeDXPrYCLGvDa4mViqCuhj-IPGhPDrlsLYG7kCfSvZ9UHMBqlZ8WDyj4zCQ3o-5AuD5tBRIqrPEmgqFO161Qu7yXjPjXvFbJM8Iu-4jf45HZy5io3OmyRZdOyfPL~Z0GkyPChoNWJax7ioZ9o6xNZu271LRecCkmoaeEBlFF~8dRZSJqXl0E9A4w2v6wXO6E8vkZioe9b8bl0wiOHVJQEDxCqzP5H9YYA__"
+        >
+          <img
+            src="./images/logo.png"
+            alt="logo Moyo"
+          />
+        </a>
```

Comment:
The logo is now correctly wrapped in a link (`<a>` tag), which addresses the previous issue. Ensure that the link is styled appropriately in your CSS.